data/README.rst

```diff
@@ -3,9 +3,9 @@ Language-Specific Data
 
 This directory contains language-specific data files. Most importantly, you will find here:
 
-1. A list of unique characters for the target language (e.g. English) in `data/alphabet.txt`
+1. A list of unique characters for the target language (e.g. English) in ``data/alphabet.txt``
```

Comment:
Is it worth mentionning the `utils/check_character.py` here to help people generate that?